api/events/schema.py

```diff
@@ -228,111 +224,65 @@ class Mutation(graphene.ObjectType):
              event[required]")
 
 
-class PaginatedDailyRoomEvents(graphene.ObjectType):
+class PaginateEvents(graphene.ObjectType):
```

Comment:
This object name is more of a verb I suggest you use a noun word like `PaginatedEvents`

Reply:
When I named the class PaginatedEvents it found a similar naming conflict in the schema so I sourced the schema for a naming design pattern and found a similarity in PaginateOffices in the schema so I for this case it can be overlooked or we suggest another name for this specific class